magento/pwa-studio
# fix: unset page loading on cms unmount

## Description

The CMS page sets a loading indicator to inform users that there is some update happening. If a user clicks away from the page fast enough the page never sets the value to `false` so the spinner gets stuck.

A simple fix, use an unmount effect to set the loading state to false.

## Related Issue





Closes [PWA-780](https://jira.corp.magento.com/browse/PWA-780).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to a category page.
2. Set network speed to "fast 3g" in dev tools.
3. Click on the Venia logo to return to home page. Note that the spinner is showing.
4. **Quickly** click on the category to go back where you came from. The spinner should not show up.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
